kitsune/wiki/views.py

```diff
@@ -1353,13 +1354,20 @@ def show_translations(request, document_slug):
     translated_locales = []
     untranslated_locales = []
 
-    translated_locales.append(document.locale)
-    translated_locales.extend(document.translations.all().values_list(
-        'locale', flat=True))
+    # This will hold locale codes only
+    translated_locales_code = []
```

Comment:
Now this line doesn't actually achieve anything, since you immediately overwrite `translated_locales_code` on line 1364.